core/foundation/src/TClassEdit.cxx

```diff
@@ -1274,15 +1284,8 @@ int TClassEdit::GetSplit(const char *type, vector<string>& output, int &nestedLo
 
 string TClassEdit::CleanType(const char *typeDesc, int mode, const char **tail)
 {
-   static const char* remove[] = {"class", "const", "volatile", nullptr};
-   auto initLengthsVector = []() {
-      std::vector<size_t> create_lengths;
-      for (int k=0; remove[k]; ++k) {
-         create_lengths.push_back(strlen(remove[k]));
-      }
-      return create_lengths;
-   };
-   static std::vector<size_t> lengths{ initLengthsVector() };
+   constexpr std::array<const char *, 3> remove{"class", "const", "volatile"};
```

Comment:
constexpr does not imply static, so that keyword is still needed

Reply:
That's true, but if it's done at compile time it doesn't matter if it's static or not I think. But yeah why not!

Reply:
I guess it depends on `-O`
https://stackoverflow.com/a/76696023/7471760
related:
https://stackoverflow.com/questions/54466591/replacing-constants-when-to-use-static-constexpr-and-inline-constexpr